build.py

```
@@ -0,0 +1,103 @@
+
+import os
+import sys
+import subprocess
+import platform
+from pathlib import Path
+
+def run_command(cmd, cwd=None, description=""):
+    """Run a command and return True if successful"""
+    try:
+        print(f"🔧 {description}")
+        result = subprocess.run(cmd, shell=True, cwd=cwd, check=True,
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)